packages/manager/tools/dev-server/src/default-config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
**thought:** Probably can live into a `configs` folder isn't it ?

Ideas:
- `./src/configs/default.js`
- `./configs/default.js`